src/renderer/actions/columns.js

```diff
@@ -0,0 +1,39 @@
+import { getDBConnByName } from './connections';
+
+
+export const FETCH_COLUMNS_REQUEST = 'FETCH_COLUMNS_REQUEST';
+export const FETCH_COLUMNS_SUCCESS = 'FETCH_COLUMNS_SUCCESS';
+export const FETCH_COLUMNS_FAILURE = 'FETCH_COLUMNS_FAILURE';
+
+
+export function fetchTableColumns (database, table) {
+  return (dispatch, getState) => {
+    if (_shouldFetchTableColumns(getState(), database, table)) {
+      return dispatch(_fetchTableColumns(database, table));
+    }
+  };
+}
+
+
+function _shouldFetchTableColumns (state, database, table) {
```

Comment:
We don't use this standard of underscore for private functions. Could you please change it?